style.css

```diff
@@ -0,0 +1,123 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: "Trebuchet MS", sans-serif;
+	font-size: 20px;
+	margin: 0;
+	padding: 0;
+	background-color: #e6f2f4;
+}
+
+nav {
+	font-size: 1.2rem;
+	height: 75px;
+	width: 100%;
+	background-color: #a7d9f4;
+	display: flex;
+	justify-content: space-around;
+	flex-direction: row;
+	align-items: center;
+}
+
+.navbar {
+	text-decoration: none;
+	color: black;
+}
+
+
+h1, h2, .me, .info, .projects {
+	text-align: center;
+	padding: 5px;
+}
+
+.personal-photo {
+	height: 300px;
+	width: 250px;
+}
+
+.photos {
+	display: flex;
+	justify-content: space-evenly;
+	padding: 10px;
+}
+
+.bball {
+	height: 300px;
+	width: 400px;
+}
+
+.leopard {
+	height: 300px;
+	width: 420px;
+}
+
+
+.box-one, .box-two {
+	width: 95%;
+	height: 200px;
+	margin-left: auto;
+	margin-right: auto;
+	margin-top: 20px;
+	margin-bottom: 20px;
+	border: 2px solid black;
+	gap: 10px;
+	position: relative;
+	text-align: center;
+}
+
+.git-link {
+	position: relative;
+	top: 100px;
```

Comment:
using non static positioning should only be done when you actually want your element to be taken out of the normal document flow, whereas here it seems you just want the github link to be at the bottom of the div. this can be achieved in other ways without using position.

in all likelihood, if you add a long description and remove the set height of the element, the github link will go where you wanted it anyway.